charts/jaeger/README.md

```diff
@@ -164,12 +201,78 @@ kubectl apply -f jaeger-tls-cfgmap.yaml
 helm install jaeger jaegertracing/jaeger --values jaeger-values.yaml
 ```
 
+### Ingester and Kafka
+
+There are additional components which can also be installed with variety of choide for Kafka similar to Cassandra and ElasticSearch
+
+![Jaeger with Ingester](images/jaeger-with-ingester.png)
+
+> Image represents agent as a sidecar and for more info to refer to [this](https://medium.com/jaegertracing/deployment-strategies-for-the-jaeger-agent-1d6f91796d09)
```

Comment:
```suggestion
```

My mistake @arpitjindal97 I took a closer look at the image, and it clearly says "host or container" meaning it represents both daemonset pod or sidecar